src/components/NavbarArrows.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Flex, Link, Button, useColorModeValue } from "@chakra-ui/react";
+import { ChevronRightIcon, ChevronLeftIcon } from "@chakra-ui/icons";
+import { useRouter } from "next/router";
+import { MotionSmall } from "./motion";
+
+interface Prop {
+  goBack: () => void;
+  goForward: () => void;
+}
+
+export default function NavbarArrows(props: Prop) {
+  let router = useRouter();
+
+  const firstValue = router.query.firstValue as string;
+
+  const secondValue = router.query.secondValue as string;
+
+  return (
+    <Box bg={useColorModeValue("gray.100", "gray.900")} px={4}>
+      <Flex h={16} alignItems="center" justifyContent="space-between">
+        <MotionSmall>
+          <Box display={["none", null, "block"]}>
+            <Button onClick={props.goBack}>
+              <ChevronLeftIcon />
+            </Button>
+          </Box>
+        </MotionSmall>
+        <MotionSmall>
```

Comment:
Given the amount of repetition, this block of code has, this would make an excellent component to refactor at the bottom of our file.